### PR TITLE
Sync Roadmap for st2 v3.2

### PR DESCRIPTION
**Pull request by armab**
Sync roadmap for upcoming st2 `v3.2` and remove already finished items from the next `v3.3`.

@StackStorm/tsc please let me know if I'm missing something or anything else we can commit in future.

### CODE REVIEW

File: docs/source/roadmap.rst [hunk 1]
```diff
@@ -65,6 +60,14 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.2
+
+* **RHEL/CentOS:** Support CentOS 8/RHEL 8 with Python 3.6 and MongoDB 4.0.
+* **Ubuntu:** Stop producing Ubuntu 14.04 packages.
```
Comment (nmaludy) [inline docs/source/roadmap.rst:66]:
```suggestion
* **Ubuntu:** Drop support for Ubuntu 14.04.
```
Thoughts?

Comment (armab) [reply docs/source/roadmap.rst:66]: We dropped support in `v3.1`, see `Done in v3.1` section.

File: docs/source/roadmap.rst [hunk 2]
```diff
@@ -16,28 +16,23 @@ contributions. Here's our plans for the next two releases.
...
+
 Monitor the `master branch <https://github.com/StackStorm/st2/>`_ to see how we're progressing.
 
 Backlog
 -------
 
-Here's some more things on our list that we haven't scheduled yet:
+Here's some more nice things on our list that we haven't scheduled yet:
 
+* **Python:** Drop Python 2 support, rely on Python 3 ony.
```
Comment (armab) [inline docs/source/roadmap.rst:33]: @blag RE `py2` vs `py3`.
There were some scattered messages, but I don't remember these kind of agreed plans for any release.
Sounds like it's a good topic for the next TSC meeting. Please raise it. I like the goal too :+1: 

Removing py 2.7 won't be that easy as it'll affect `U16` and `EL7` and packaging we currently use.
Probably worth a more in-depth technical @StackStorm/tsc conversation what's technically involved there and commitments around this item.

File: docs/source/roadmap.rst (hunk 1 above)
Comment (blag) [reply docs/source/roadmap.rst:66]: We also dropped all testing of the Ubuntu 14.04 packages. Ideally, every packaged environment is also tested and officially supported, but that's not always the case, and it wasn't the case here.

I don't know how relevant that will be to any/many users though.

File: docs/source/roadmap.rst (hunk 2 above)
Comment (blag) [reply docs/source/roadmap.rst:33]: Gotcha, that makes sense. I'll add it to the docket for the next TSC meeting.

File: docs/source/roadmap.rst (hunk 1 above)
Comment (armab) [reply docs/source/roadmap.rst:66]: To recap. It's possible to stop supporting the OS/distro, while still shipping the packages. This means no support, help, bugfixes provided as well as no issues accepted related to that distro/installation.

We added U18 and dropped U14 support in `v3.1`. By shipping the U14 packages one more release after `v3.1` we just gave some time to someone who was still relying on that but couldn't switch to other supported OS/distro.

And so as result Roadmap resulted in the following:
```
## Done in v3.2
**Ubuntu:** Stop producing Ubuntu 14.04 packages.

## Done in v3.1
**Ubuntu:** Drop Ubuntu 14.04 support. Packages are still available for a limited time.
**Ubuntu:** GA Support Ubuntu 18.04, with Python 3.6
```

@nmaludy I hope that makes more sense now.

Comment (nmaludy) [reply docs/source/roadmap.rst:66]: Awesome!

**Comment by nmaludy** [comment]
@armab FYI i just rebased this with master for you

File: docs/source/roadmap.rst [hunk 3]
```diff
@@ -16,28 +16,24 @@ contributions. Here's our plans for the next two releases.
...
+* **ChatOps:** Support RBAC.
+* **Ubuntu:** Support Ubuntu 20.04 LTS.
```
Comment (armab) [inline docs/source/roadmap.rst:35]: @blag Following your idea, just included Ubuntu 20.04 LTS to the list of items that's not assigned to any release yet.

Comment (blag) [reply docs/source/roadmap.rst:35]: Awesome, thanks!

**Pull request merged**